tools/smarterr-parser/main.go

```diff
@@ -0,0 +1,299 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Perhaps include a make target to build and install this (similar to `make skaff`)?